src/js/app-es6/controllers/PontoController.js

```diff
@@ -56,32 +56,52 @@ class PontoController {
 
         let msg;
 
-            if (this._listaPontos.pontos.length == 0) {
-                msg = 'Lista já está vazia'
+        if (this._listaPontos.pontos.length == 0) {
+            msg = 'Lista já está vazia'
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)